CONTRIBUTING.md

```diff
@@ -77,25 +77,28 @@ requested:
 After the PR has been reviewed it is the job of the reviewer to merge the PR.
 Instructions for this are given below.
 
-### How we review
+### Life of a PR
 
 We try to review PRs at least once per 24h window.
 If your PR remains untouched for more than 24h, please ping us.
 Anything will do "PTAL", "up".
 Sometimes, we are on holiday, but you cannot guess, so in doubt, ping us 😊.
 
-Maintainers don't own specific parts of the project. We try to use our best
-judgment when doing reviews.
-For small PRs (trivial fix, or additions), we consider 1 reviewer is enough,
-and can merge the PR directly after the review.
+When a maintainers sees an unassigned PR, they have 2 options:
 
-For larger PRs and non-trivial changes, 2 pair of eyes are always good.
+- If it's a small & trivial change, they will self-assign, and handle
+the approval/merging.
+- Otherwise, they will chose 2 or more reviewers (can self-assign).
+- If a maintainer wants to review, they can add themselves to the reviewer list.
+- If a maintainer comments without adding a review, they don't block merging.
 
-Between maintainers, we try to follow the following convention:
- - if a maintainer approves a PR, the others are free to merge if they
-approve themselves.
- - if a maintainer requests changes without approval, they want to look at
-it again once the changes are in.
+When a change has multiple reviewers, we try to follow the following rule:
```

Comment:
I think this means reviewers do the merge, and all reviewers must approve?

Reply:
Yes, rephrased a bit to make it clear :)